client/public/index.html

```diff
@@ -47,8 +47,6 @@
       href="https://fonts.googleapis.com/icon?family=Material+Icons"
       rel="stylesheet"
     />
-    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/css/materialize.min.css">
```

Comment:
don't know if it is a problem but I'm adding this commentary just in case i need to retrieve it

Reply:
hopefully it won't affect the overall project